ref/gl/gl_decals.c

```diff
@@ -811,15 +811,27 @@ void R_DecalShoot( int textureIndex, int entityIndex, int modelIndex, vec3_t pos
 	decalInfo.m_Flags = flags;
 
 	R_GetDecalDimensions( textureIndex, &width, &height );
-	decalInfo.m_Size = width >> 1;
```

Comment:
This can be done completely on engine side. See `CL_PlayerDecal` function in the engine. I think CL_DecalShoot can be replaced here with direct RefAPI call, adjusting scale according to `cl_logomaxdim`.

Reply:
No, because other stuff like `m_decalWidth`/`m_decalHeight`/`m_Size` should be scaled too.

Reply:
I'm not particularly fond of decal code right for a multitude of reasons, one of which is #63. 

I would rather not touch it and only pass stuff through the engine right now, so we could rewrite (and possibly fix, it is also buggy as fuck) it later.

Reply:
> pngquant for example does an excellent job recompressing stuff most of the time.

In my experience - zopflipng is better recompressor for png.

Reply:
> Explicitly, no. Maybe GIMP includes it and uses under the hood, I was using maximum compression level for exporting PNG in there. Anyway, problem with all such external tools it that almost nobody uses something except generic photo editors like Paint/GIMP/Photoshop/built-in OS photo editor.

All image editors uses default filter and default deflate's window size.